lib/main.dart

```diff
@@ -99,6 +104,11 @@ class _MyHomePageState extends State<MyHomePage> {
     });
   }
 
+  @override
+  void initState() {
+    super.initState();
```

Comment:
is this initState() needed since we are only calling super.initState()?

Reply:
Yep, this should be removed.